test/core/runtime/runtime_external_interface_test.cpp

```diff
@@ -136,6 +137,7 @@ class REITest : public ::testing::Test {
       "  (type (;30;) (func (param i32 i64) (result i32)))\n"
       "  (type (;31;) (func (param i32 i32 i64) (result i64)))\n"
       "  (type (;32;) (func (param i32 i64 i32) (result i32)))\n"
+      "  (type (;33;) (func (param i64 i64 i32)))\n"
```

Comment:
In according of spec:
```suggestion
      "  (type (;33;) (func (param i64 i64 i32) (result i64)))\n"
```